grpc/job.hpp

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+namespace ioremap { namespace grpc {
+
+class job_t {
```

Comment:
Please, put comments here and in all "jobs"

Reply:
553fb9f